lib/driver.js

```diff
@@ -473,10 +473,7 @@ class AndroidDriver extends BaseDriver {
   }
 
   getPrefsBuilder () {
```

Comment:
sure, maybe @imurchie knows how to mock `new SharedPrefsBuilder` with sinon. I tried some implementations but none of them worked 😞